Formula/lib/libkeyfinder.rb

```diff
@@ -0,0 +1,32 @@
+class Libkeyfinder < Formula
+  desc "Musical key detection for digital audio, GPL v3"
+  homepage "https://mixxxdj.github.io/libkeyfinder/"
+  url "https://github.com/mixxxdj/libkeyfinder/archive/refs/tags/2.2.8.tar.gz"
+  sha256 "a54fc6c5ff435bb4b447f175bc97f9081fb5abf0edd5d125e6f5215c8fff4d11"
+  license "GPL-3.0-only"
```

Comment:
should be `GPL-3.0-or-later`?
```suggestion
  license "GPL-3.0-or-later"
```

```
/*************************************************************************

  Copyright 2011-2015 Ibrahim Sha'ath

  This file is part of LibKeyFinder.

  LibKeyFinder is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, either version 3 of the License, or
  (at your option) any later version.

  LibKeyFinder is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License
  along with LibKeyFinder.  If not, see <http://www.gnu.org/licenses/>.

*************************************************************************/
```